### PR TITLE
Feat: create the definition for built-in policies

### LINKED ISSUE #3879: [Feature] create the definition for built-in policies
**Is your feature request related to a problem? Please describe.**
Currently, there are no definitions for built-in policies, so, we can not get the API schema and UI schema. This led to we can not generate the Doc and can not configure the policy in VelaUX.

**Describe the solution you'd like**
Create the definition for built-in policies, they only are used to generate API schema and UI schema.

**Additional context**

built-in policies include:

* override
* topology
* env-binding(deprecated soon)

### PR DESCRIPTION
**Pull request by ZhongsJie96**
Signed-off-by: Shijie Zhong <zhongsjie@cmbchina.com>

### Description of your changes

create the definition for built-in policies

Fixes #3879 

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by fourierr — APPROVED** [review]

**Review by barnettZQG — APPROVED** [review]
LGTM

**Comment by wonderflow** [comment]
@ZhongsJie96 Are these definitions align parameters with the inner Implementation? Is it possible to have a mechanism to keep they're consistent.

File: vela-templates/definitions/internal/policy/override.cue [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+template: {
+
+	#PatchParams: {
+		// +usage=Specify the name of the patch component, if empty, all components will be merged
+		name?: string
+		// +usage=Specify the type of the patch component.
+		type?: string
+		properties?: {...}
+		triats?: [...{
```
Comment (Somefive) [inline vela-templates/definitions/internal/policy/override.cue:17]: typo here. triats -> traits

File: vela-templates/definitions/internal/policy/topology.cue [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+template: {
+	parameter: {
+		// +usage=Specify the names of the clusters to select.
+		cluster?: [...string]
+		// +usage=Specify the label selector for clusters
+		clusterLabelSelector?: [string]: string
+		// +usage=Deprecated: Use clusterLabelSelector instead.
+		deprecatedClusterSelector?: [string]: string
```
Comment (Somefive) [inline vela-templates/definitions/internal/policy/topology.cue:16]: deprecatedClusterSelector -> clusterSelector

Comment (ZhongsJie96) [reply vela-templates/definitions/internal/policy/topology.cue:16]: ok, I will modify it

File: vela-templates/definitions/internal/policy/override.cue (hunk 1 above)
Comment (ZhongsJie96) [reply vela-templates/definitions/internal/policy/override.cue:17]: Sorry! This is a typo error

**Comment by ZhongsJie96** [comment]
> @ZhongsJie96 Are these definitions align parameters with the inner Implementation? Is it possible to have a mechanism to keep they're consistent. I'm afrait next time when we update the Implementation with some changes and these definitions probably be forgotten.

The definition of policy is mainly based on the specific implementation, including parameters that can be omitted. If there are changes for implementation in the later stage, I think the change involving the template is an inevitable problem.

**Review by wonderflow — APPROVED** [review]

**Pull request merged**